packages/switch/test/lion-button-switch.test.js

```diff
@@ -0,0 +1,146 @@
+import { expect, fixture, html } from '@open-wc/testing';
+import sinon from 'sinon';
+
+import { LionButton } from '@lion/button';
+
+import '../lion-button-switch.js';
+
+describe('lion-button-switch', () => {
+  let el;
+  beforeEach(async () => {
+    el = await fixture(html`
+      <lion-button-switch></lion-button-switch>
+    `);
+  });
+
+  it('should behave as a button', () => {
+    expect(el instanceof LionButton).to.be.true;
+  });
+
+  it('should have tabindex="0"', () => {
+    expect(el.tabIndex).to.equal(0);
+    expect(el.getAttribute('tabindex')).to.equal('0');
+  });
+
+  it('should have type="button"', () => {
+    expect(el.type).to.equal('button');
+    expect(el.getAttribute('type')).to.equal('button');
+  });
+
+  it('should have disabled=false by default', () => {
+    expect(el.disabled).to.equal(false);
+    expect(el.hasAttribute('disabled')).to.be.false;
+  });
+
+  it('should have checked=false by default', () => {
```

Comment:
I think this one makes sense to test :), because it's a core feature that is toggleable so default state here is kinda important to show as "behavior"